<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectExplorer::Internal::CustomParserConfigDialog</class>
 <widget class="QDialog" name="ProjectExplorer::Internal::CustomParserConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom Parser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabError">
      <attribute name="title">
       <string>Error</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>&amp;Error message capture pattern:</string>
           </property>
           <property name="buddy">
            <cstring>errorPattern</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="errorPattern">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="errorCapturePositionsGroup">
         <property name="title">
          <string>Capture Positions</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&amp;File name:</string>
            </property>
            <property name="buddy">
             <cstring>errorFileNameCap</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&amp;Line number:</string>
            </property>
            <property name="buddy">
             <cstring>errorLineNumberCap</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&amp;Message:</string>
            </property>
            <property name="buddy">
             <cstring>errorMessageCap</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="errorFileNameCap">
            <property name="maximum">
             <number>9</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="errorLineNumberCap">
            <property name="maximum">
             <number>9</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="errorMessageCap">
            <property name="maximum">
             <number>9</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="errorCaptureChannelsGroup">
         <property name="title">
          <string>Capture Output Channels</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="errorStdOutChannel">
            <property name="text">
             <string>Standard output</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="errorStdErrChannel">
            <property name="text">
             <string>Standard error</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="errorTestGroup">
         <property name="title">
          <string>Test</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>E&amp;rror message:</string>
            </property>
            <property name="buddy">
             <cstring>errorOutputMessage</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="errorOutputMessage">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>File name:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="errorFileNameTest">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Line number:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="errorLineNumberTest">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Message:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="errorMessageTest">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabWarning">
      <attribute name="title">
       <string>Warning</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Warning message capture pattern:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="warningPattern">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="warningCapturePositionsGroup">
         <property name="title">
          <string>Capture Positions</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="2">
           <widget class="QSpinBox" name="warningLineNumberCap">
            <property name="maximum">
             <number>9</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QSpinBox" name="warningMessageCap">
            <property name="maximum">
             <number>9</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>&amp;Line number:</string>
            </property>
            <property name="buddy">
             <cstring>errorLineNumberCap</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>&amp;File name:</string>
            </property>
            <property name="buddy">
             <cstring>errorFileNameCap</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QSpinBox" name="warningFileNameCap">
            <property name="maximum">
             <number>9</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>&amp;Message:</string>
            </property>
            <property name="buddy">
             <cstring>errorMessageCap</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="warningOutputChannelsGroup">
         <property name="title">
          <string>Capture Output Channels</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="warningStdOutChannel">
            <property name="text">
             <string>Standard output</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="warningStdErrChannel">
            <property name="text">
             <string>Standard error</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="warningTestGroup">
         <property name="title">
          <string>Test</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Warning message:</string>
            </property>
            <property name="buddy">
             <cstring>errorOutputMessage</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="warningOutputMessage">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>File name:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="warningFileNameTest">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Line number:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="warningLineNumberTest">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Message:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="warningMessageTest">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>errorPattern</tabstop>
  <tabstop>errorFileNameCap</tabstop>
  <tabstop>errorLineNumberCap</tabstop>
  <tabstop>errorMessageCap</tabstop>
  <tabstop>errorStdOutChannel</tabstop>
  <tabstop>errorStdErrChannel</tabstop>
  <tabstop>errorOutputMessage</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>warningPattern</tabstop>
  <tabstop>warningFileNameCap</tabstop>
  <tabstop>warningLineNumberCap</tabstop>
  <tabstop>warningMessageCap</tabstop>
  <tabstop>warningStdOutChannel</tabstop>
  <tabstop>warningStdErrChannel</tabstop>
  <tabstop>warningOutputMessage</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectExplorer::Internal::CustomParserConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectExplorer::Internal::CustomParserConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
